charts/bpdm/values.yaml

```diff
@@ -0,0 +1,81 @@
+bpdm-gate:
+  enabled: true
+  replicaCount: 1
+  image:
+    registry: ghcr.io
+    repository: catenax-ng/tx-bpdm/gate
+    pullPolicy: Always
+    tag: ""
+  # ... more configuration for bpdm-gate if need...
+  applicationConfig:
+    bpdm:
+      pool:
+        base-url: ""
+
+bpdm-pool:
```

Comment:
I know due to limitations of helm currently child of child dependencies are ignored. But I feel for being more clear here we should still disable postgres and opensearch dependencies here by setting enabled: false